shared/js/email.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Sends the tracker metrics to the user via Textile pass-through API
+ *
+ * This will be browserifyed and turned into email.js by running 'grunt'
+ */
+
+require.scopes.email = ( () => {
+
+    const ONEDAY = 1000 * 60 * 60 * 24;
+
+    function sendMetrics() {
+
+        chrome.storage.local.get(function (result) {
+
+            var payload = {
+                companyData: result.companyData,
+                totalPages: result.totalPages,
+                totalPagesWithTrackers: result.totalPagesWithTrackers
+            }
```

Comment:
@andrewxhill @asutula we could also add the info from 'site.js`, check the info in the `Score` class. that might get unwieldy pretty fast as info we want to include in the email, but perhaps we could send off the _really_ low score sites (letter grade D)